src/main/java/com/cx/restclient/sast/dto/SASTResults.java

```diff
@@ -323,33 +448,49 @@ private void setScanStartEndDates(String scanStart, String scanTime) {
 
     }
 
-    private String formatToDisplayDate(Date date) {
-        //"26/2/17 12:17"
-        String displayDatePattern = "dd/MM/yy HH:mm";
-        Locale locale = Locale.ENGLISH;
-
-        return new SimpleDateFormat(displayDatePattern, locale).format(date);
+    private String formatToDisplayDate(Date date) throws ParseException{
+    	 String displayDatePattern = "dd/MM/yy HH:mm";
+         Locale locale = Locale.ENGLISH;
+         return new SimpleDateFormat(displayDatePattern, locale).format(date);
     }
 
+    
+    
     private Date createStartDate(String scanStart) throws Exception {
         DateFormat formatter;
         Date formattedDate = null;
-
-        for (SupportedLanguage lang : SupportedLanguage.values()) {
-            try {
-                formatter = new SimpleDateFormat(lang.getFormat(), lang.getLocale());
-                formattedDate = formatter.parse(scanStart);
-                break;
-            } catch (Exception ignored) {
-            }
-        }
-
+			try {
+				//for chinese simplified parsing
+				if(this.language.equalsIgnoreCase("zh-CN")) {
```

Comment:
Not the best way to write this using if...else.   You could obtain format to be used based on the enumeration like it was discussed.